index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <main>
+      <div class="jumbotron header">
+        <h1 id="title" >AYO MOSES ADEYEMI</h1>
+        <p>The Man Who Won A Million Souls To Christ!!!!</p>
+      </div>
+      <div id="img-div">
+        <img id="image" src="https://cdn.britannica.com/43/220943-138-FAFEB049/did-you-know-Martin-Luther-King-Jr.jpg?w=800&h=450&c=crop" alt="">
```

Comment:
Hi Ayo. I love the nice pics you used here. I will like to suggest that you also include an alt attribute to your image tag to assist screen readers in knowing what your wonderful image is all about